backend/tracim_backend/models/reaction.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy import Sequence
+from sqlalchemy import Unicode
+from sqlalchemy import UniqueConstraint
+from sqlalchemy.orm import relationship
+from sqlalchemy.types import Integer
+
+from tracim_backend.models.auth import User
+from tracim_backend.models.data import Content
+from tracim_backend.models.meta import DeclarativeBase
+from tracim_backend.models.mixins import CreationDateMixin
+
+
+class Reaction(CreationDateMixin, DeclarativeBase):
```

Comment:
Aren't you missing the unique condition on (content_id, value, author_id)?

Reply:
No really, check line 17 (2 line below):
```python
__table_args__ = (UniqueConstraint("author_id", "content_id", "value"),)
```